tests/contrib/logging/test_logging.py

```diff
@@ -158,3 +163,26 @@ def create_span():
 
         with self.override_global_config(dict(version="global.version", env="global.env")):
             self._test_logging(create_span=create_span, version="global.version", env="global.env")
+
+    def test_unfinished_child(self):
+        """
+        Check that closing a span with unfinished children correctly logs out
+        unfinished spans. See #1337 for more context.
```

Comment:
We can skip adding the issue number here as it would be redundant with this PR.

```suggestion
        unfinished spans.
```